resources/prometheus-ro-rootfs.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
```

Comment:
Does that work without `[Install]`? Do we run it anywhere already so I can see it in action?

Reply:
This service is not "installed". It is started by the timer, that by default starts a unit with the same name but being .service. It is similar to the behaviour of the systemd_service_restarter module.